metric-example/metric-expressions-by-example.md

```diff
@@ -0,0 +1,248 @@
+# Metric Expressions by Example
+This is an example-oriented document that will teach you advanced 
+techniques with metric selectors that will enable you to:
+* convert °F to Kelvin,
+* calculate an error rate metric from a service call count metric and an error count metric,
+* show the available memory of those hosts that run the most memory-intensive process group instances.
+
+## Prerequisites
+This document assumes basic familiarity with metric selectors. If you don't yet feel comfortable with simple metric selectors, consider reading
+through [Query by Example](query-by-example.md) first.
+
+You can try the examples listed here through the Web UI via _Data Explorer_,
+or you can send selectors to the `/v2/metrics/query` API for evaluation.
+
+## Scenario 1: Convert Fahrenheit to Kelvin
+> **Task**
```

Comment:
After this occurrence, "**Task**" is always on the same line as the following text.

Reply:
...except Scenario 4

Reply:
Let's make it consistent by having it always on the same line.